app/src/main/res/values/strings.xml

```diff
@@ -4,12 +4,34 @@
     <string name="google">Login With Google</string>
     <string name="or">OR</string>
     <string name="phone">Enter Your Phone Number</string>
-    <string name="login">Login</string>
+
     <string name="sso">Login with SSO</string>
     <string name="forgot">Forgot Password ?</string>
     <string name="signup">No account yet ? Sign Up</string>
     <string name="download">Looking for the app ? Download Here</string>
     <string name="pass">Enter Paytm Password</string>
     <string name="verify">Verify Your Account</string>
     <string name="proceed">Proceed Securely</string>
+    
+    <string name="Hindi">Hindi</string>
+    <string name="skip">Skip</string>
+    <string name="Login">Login or Create an Account</string>
+    <string name="help">Need Help ?</string>
+    <string name="info">For verification, an SMS will be sent from your mobile number. Standard operator charges may apply. By proceeding you agree to our Terms and Conditions and Privacy Policy.</string>
```

Comment:
@raghav32 add suitable key name for strings
such as for this you can mention whether it is a label, header or subhead and write keyName as 
lbl_verification_info / head_verification_info / subhead_verification_info

Please fix this for all strings. 
Also remove strings which are not being used